test/src/pelz_test.c

```diff
@@ -28,11 +31,33 @@ int clean_suite(void)
 //Main function for the unit testing of the Pelz Service application
 int main(int argc, char **argv)
 {
+  char cwd[1024];
+  char *tmp_id;
+  CharBuf id;
+  char *key[6] = { "KIENJCDNHVIJERLMALIDFEKIUFDALJFG", "KALIENGVBIZSAIXKDNRUEHFMDDUHVKAN", "HVIJERLMALIDFKDN",
+		   "NGVBIZSAIXKDNRUE", "EKIUFDALVBIZSAIXKDNRUEHV", "ALIENGVBCDNHVIJESAIXEKIU"};
+
   set_app_name("pelz");
   set_app_version("0.0.0");
   set_applog_path("./test/log/pelz.log");
   set_applog_severity_threshold(LOG_INFO);
 
+  getcwd(cwd, sizeof(cwd));
+  tmp_id = "/test/key1.txt";
+  id = newCharBuf(strlen(tmp_id) + strlen(cwd));
+  memcpy(id.chars, cwd, strlen(cwd));
+  memcpy(&id.chars[strlen(cwd)], tmp_id, (id.len - strlen(cwd)));
+  for (int i = 1; i < 7; i++)
```

Comment:
It would be clearer what's going on here if the length of the `key` array was a variable and that same variable got used for this loop.